### PR TITLE
Fix CI for 1.8.8

### PR DESCRIPTION
**Pull request by adam-mesos**
Bumps dcos-vagrant, pins webtest-aiohttp, and sets exhibitor's JAVA_HOME to get the 1.8.x CI green again.

### CODE REVIEW

**Comment by adam-mesos** [comment]
@cmaloney @orsenthil @mellenburg @lingmann 
I cherry-picked a couple of things from master and got the build+tests green, but my direct attempts to fix the pep8 errors resulted in broken builds. I tried cherry-picking the flake8 plugin, but that just complains about more style errors.
I wonder if it's worth merging this in before we fix the pep8 errors to unblock other 1.8.8 PRs, since we can tell when we're introducing new style errors.

**Comment by orsenthil** [comment]
@adam-mesos , the codequality test will be fixed by back-porting this change: https://github.com/dcos/dcos/pull/1114

**Comment by orsenthil** [comment]
@adam-mesos , the other code-quality failures are real and those needs to be fixed. https://teamcity.mesosphere.io/viewLog.html?buildId=529313

**Comment by adam-mesos** [comment]
@orsenthil Maybe you can help me figure out why adding blank lines in https://github.com/dcos/dcos/pull/1101/commits/568b557faead0abb5254e5e7e4318dca13d98c05 breaks the `create-release-pr` step, where the previous commit fails the `code-quality` check, but builds and passes the other tests.

**Comment by orsenthil** [comment]
So, the new problem is, exhibiter is unable to get a plugin that it depends upon and the grade build fails.

```
[09:04:27]	Checking connectivity... done.
[09:04:37]	
[09:04:37]	FAILURE: Build failed with an exception.
[09:04:37]	
[09:04:37]	* Where:
[09:04:37]	Build file '/pkg/src/exhibitor/build.gradle' line: 2
[09:04:37]	
[09:04:37]	* What went wrong:
[09:04:37]	Error resolving plugin [id: 'nebula.netflixoss', version: '2.2.5']
[09:04:37]	> Could not GET 'https://plugins.gradle.org/api/gradle/2.2.1/plugin/use/nebula.netflixoss/2.2.5'.
[09:04:37]	   > peer not authenticated
[09:04:37]
```

* I am figuring out this is happening of a sudden. We have java security certs in our build agents (otherwise it wouldn't have worked at all)
* The plugin's certs had not been updated recently and are valid.

**Comment by orsenthil** [comment]
@BenWhitehead hurray and thanks.
@adam-mesos good news. :)

**Comment by adam-mesos** [comment]
@orsenthil @BenWhitehead @lingmann It's all green! :green_heart: Please review.
Testing 1.8.8 EE in https://github.com/mesosphere/dcos-enterprise/pull/480

File: packages/exhibitor/build [hunk 1]
```diff
@@ -10,6 +10,9 @@ rm -rf "$PKG_PATH/usr/zookeeper/"{src,docs,contrib}
 mkdir -p "$PKG_PATH/bin"
 ln -s "$PKG_PATH/usr/zookeeper/bin/zkCli.sh" "$PKG_PATH/bin/zkCli.sh"
 
+# Explicitly set JAVA_HOME to our java package
+export JAVA_HOME="$(find /opt/mesosphere/packages -type d -name java--*)/usr/java"
```
Comment (lingmann) [inline packages/exhibitor/build]: I noticed that this is not on master, and I suspect we want it there as well... @BenWhitehead wdyt?

Comment (BenWhitehead) [inline packages/exhibitor/build]: Cherry picked to another PR targeting master: https://github.com/dcos/dcos/pull/1130

**Comment by BenWhitehead** [comment]
I've rebased to remove the python 3.4 -> 3.5 changes, if this builds cleanly we'll proceed with this. If it doesn't I've got a local backup I can re-push to revert this rebase.

**Comment by BenWhitehead** [comment]
@adam-mesos We're all green with py34!

@lingmann @orsenthil Can We get a couple reviews on this?

**Comment by lingmann** [comment]
Cool I'm a 🚢 
@orsenthil we just need your review now.

**Comment by orsenthil** [comment]
@lingmann  done! 👍

**Pull request merged**